src/tree/updater_quantile_hist.h

```diff
@@ -148,6 +149,40 @@ struct CPUHistMakerTrainParam
   }
 };
 
+/* tree growing policies */
+struct ExpandEntry {
```

Comment:
Do we need to worry about name colision?  Possibly during link time?

Reply:
@ShvetsKS I think this is a possible cause of the issue.

Reply:
@trivialfis  yes! Thanks for noticing it. I'll prepare fix